views/index.html

```diff
@@ -18,27 +18,27 @@
           </div>
           <h3>Product Development</h3>
             <p>
-              We build <i>mobile-first</i> MVPs with intuitive and user-friendly interfaces.
+              We build responsive, mobile-first MVPs with intuitive and human-centred design.
             </p>
         </div>
       </div>
       <div class="pure-u-1 pure-u-md-1-3 flex-center section-box-wrapper">
         <div class="section-box">
           <div class="offering-icon user-testing-icon">
           </div>
-          <h3>User testing</h3>
+          <h3>Co-working</h3>
             <p>
-              We mentor you on user testing, customer development and product ownership.
+              We curate <a href='https://www.focushub.co.uk/'>productive workspaces</a> for freelancers, non-profits and small businesses.
             </p>
         </div>
       </div>
       <div class="pure-u-1 pure-u-md-1-3 flex-center section-box-wrapper">
         <div class="section-box">
           <div class="offering-icon training-icon">
           </div>
-          <h3>Training</h3>
+          <h3>Technical Training</h3>
             <p>
-              We provide professional training for developers in TDD, Node.js and React.
+              Hands-on, immersive learning designed to enlighten, inspire and empower. Workshops in agile development, product ownership, human-centered design and more.
```

Comment:
Took me 2 mins to pull this branch, `npm i && npm start`. Always worth running the code you're changing.

<img width="1192" alt="screen shot 2017-06-01 at 16 34 08" src="https://cloud.githubusercontent.com/assets/4185328/26687642/300315e0-46e8-11e7-890a-e1540bd74052.png">